code/controllers/master.dm

```diff
@@ -641,19 +641,18 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 	return tab_data
 
 /datum/controller/master/StartLoadingMap()
-	//disallow more than one map to load at once, multithreading it will just cause race conditions
-	while(map_loading)
-		stoplag()
-	for(var/S in subsystems)
-		var/datum/controller/subsystem/SS = S
-		SS.StartLoadingMap()
-	map_loading = TRUE
+	map_loading ++
+	if (map_loading == 1)
```

Comment:
please define those

Reply:
This number represents the number 1, as 1 map is being loaded. I don't see the need for a define since the number isn't magic and it speaks for itself

Reply:
it doesn't speak for itself